src/models/Expense.model.js

```diff
@@ -1,9 +1,45 @@
 'use strict';
 
+const { DataTypes } = require('sequelize');
 const { sequelize } = require('../db.js');
 
 const Expense = sequelize.define(
-  // your code goes here
+  'Expense',
+  {
+    id: {
+      type: DataTypes.INTEGER,
+      primaryKey: true,
+    },
```

Comment:
Set `autoIncrement` and you don't have to generate ids manually
https://sequelize.org/docs/v6/core-concepts/model-basics/
![image](https://github.com/mate-academy/node_accounting-app-with-db/assets/63243136/cf2217ce-5c98-40dd-a3a0-c81af4a6d9f0)